src/test/java/com/team701/buddymatcher/controllers/users/ValidateControllerIntegrationTest.java

```diff
@@ -0,0 +1,57 @@
+package com.team701.buddymatcher.controllers.users;
+
+import com.team701.buddymatcher.config.JwtTokenUtil;
+import com.team701.buddymatcher.domain.users.User;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.web.servlet.MockMvc;
+
+import java.util.Random;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT, properties = {
+        "socketio.host=localhost",
+        "socketio.port=8085"
```

Comment:
```suggestion
        "socketio.port=8087"
```
Use a different port just in case. My experience using the same port is it could cause flaky test as socketio client doesn't close fast enough

Reply:
Thanks for the suggestion!